text/work_packages_and_reports.tex

```diff
@@ -149,50 +154,56 @@ \subsection{Computation of Work Results}\label{sec:computeworkresult}
     (\mathbb{P}, \N_\mathsf{C}) &\to \mathbb{W} \\
     (\wpX, c) &\mapsto \begin{cases}
         \error &\when \mathbf{o} \not\in \Y \\
-        \tup{\is{a}{\wpX_\wp¬authorizer}, \mathbf{o}, \is{x}{\wpX_\wp¬context}, s, \mathbf{r}} \!\!\!\!\!&\otherwise
+        \tup{\is{a}{\wpX_\wp¬authorizer}, \mathbf{o}, \is{x}{\wpX_\wp¬context}, s, \mathbf{l}, \mathbf{r}} \!\!\!\!\!&\otherwise
     \end{cases}
   \end{aligned}\right.\!\!\!\!\!\!\!
 \end{equation}
 
-where:
+Where:
 \begin{align*}
-  \mathbf{o} &= \Psi_I(\wpX, c) \\
-  (\mathbf{r}, \overline{\mathbf{e}}) &= \transpose[
+  \keys{\mathbf{l}} \equiv \,&\{ h \mid w \in p_\mathbf{w}, (h^\boxplus, n) \in w_\mathbf{i} \} \ ,\quad|\mathbf{l}| \le 8\\
+  \mathbf{o} = \,&\Psi_I(\wpX, c) \\
+  (\mathbf{r}, \overline{\mathbf{e}}) = \,&\transpose[
     (\itemtoresult(\wpX_\wp¬workitems[j], r), \mathbf{e})
     \mid
     (r, \mathbf{e}) = \countupexports(\wpX, j),
     j \orderedin \N_{|\wpX_\wp¬workitems|}
   ] \\
-  \countupexports(\wpX, j) &\equiv \marshallrefine(
-    \wpX,
-    \wpX_\wp¬workitems[j],
-    \sum_{k < j}\wpX_\wp¬workitems[k]_\wi¬exportsegments
-  )\\
-  \marshallrefine(\wpX, w, \ell) &\equiv \\
-  &\!\!\!\!\!\!\!\Psi_R(
+  \countupexports(\wpX, j) \equiv \,&\Psi_R(
     \wiX_c,
     \wiX_g,
     \wiX_s,
-    \mathcal{H}(\wpX),
+    h,
     \wiX_\mathbf{y},
     \wpX_\wp¬context,
```

Comment:
Think there is an argument missing here as Psi_R takes 11 arguments AFAICT, but only 10 are passed here. Think the authorizer output o is what's missing?

Reply:
I'll put it in